app/serializers/current_user_serializer.rb

```diff
@@ -43,6 +43,7 @@ class CurrentUserSerializer < BasicUserSerializer
              :top_category_ids,
              :hide_profile_and_presence,
              :groups,
+             :owned_groups,
```

Comment:
I am with @tgxworld  here I think I would like one less query by default, so we need to feed this information in on demand, vast majority of the time the client does not need to keep track of which groups are owned by current user vs not. (note when sending them down we should use ids anyway)